app/controllers/WeHaveConfirmedYourIdentityController.scala

```diff
@@ -46,20 +48,21 @@ class WeHaveConfirmedYourIdentityController @Inject() (
   def onPageLoad: Action[AnyContent] = (identify andThen getData.apply andThen requireData).async {
 
     implicit request =>
-      matchInfo flatMap {
-        // TODO implement redirection logic
-        case _ => renderer.render("weHaveConfirmedYourIdentity.njk").map(Ok(_))
+      matchIndividualInfo flatMap {
+        case Right(_) =>
+          renderer.render("weHaveConfirmedYourIdentity.njk").map(Ok(_))
+        case _ =>
+          request.userAnswers.set(NotMatchingInfoPage, "identity").foreach(sessionRepository.set)
```

Comment:
@iservport Instead of hardcoding, NotMatchingInfoPage can we have 2 different controllers one for not matched Business and one for not matched individual